docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
     volumes:
       - db_data:/var/lib/postgresql/data
     environment:
+      POSTGRES_DB: "cal-saml"
```

Comment:
Will this affect self-hosted users that don't have SAML enabled?

Reply:
Shouldn't matter too much other than creation of the DB. Alternative is to maintain another copy of docker-compose which seems an overkill.

Reply:
Actually POSTGRES_DB can be passed in as an env var using the dotenv files

Reply:
Actually that won't work, docker-compose doesn't pick up the dotenv files.